src/smt/term_formula_removal.h

```diff
@@ -166,7 +166,7 @@ class RemoveTermFormulas {
   static bool hasNestedTermChildren( TNode node );
 
   /** A proof node manager */
-  std::unique_ptr<ProofNodeManager> d_pnm;
+  ProofNodeManager* d_pnm;
```

Comment:
why did you change this?

Reply:
Because we will use a single global proof node manager, objects should not create their own.